certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2934 @@
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Existing resources returned",
+            "schema": {
+              "$ref": "#/definitions/ResourceResponseListResponse"
+            }
+          },
+          "401": {
+            "description": "User not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "Operation not allowed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Resources"
+        ],
+        "summary": "Create a Resource",
+        "description": "Creates a Resource",
+        "operationId": "ResourcesCreate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/CreateResourceRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "New resource returned",
+            "schema": {
+              "$ref": "#/definitions/ResourceResponseResponse"
+            }
+          },
+          "400": {
+            "description": "Resource not valid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "User not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "Operation not allowed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/Webhooks/{id}": {
+      "get": {
+        "tags": [
+          "Webhooks"
+        ],
+        "summary": "Get a Webhook",
+        "description": "Gets a Webhook",
+        "operationId": "Webhooks_Get",
+        "x-ms-visibility": "internal",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "description": "Existing Webhook Id",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Webhook Id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns existing item",
+            "schema": {
+              "$ref": "#/definitions/WebhookResponseResponse"
+            },
+            "examples": {
+              "application/json": {
+                "data": {
+                  "id": "d021c105-fc11-4cec-b45d-d6eba05ef410",

```

Comment:
@vsorokin-ppmexpress please replace this secret-look-a-like value with a string like in other examples.